### PR TITLE
UI fixes for Collapse, SocialButton, Popover and Separator

### PR DESCRIPTION
**Pull request by silvenon**
UI fixes for:

- Collapse ([ORBIT-1979](https://jira.kiwi.com/browse/ORBIT-1979))
  - the problem was that `:focus` styles in ButtonPrimitve were overriding `:hover` styles, so I did some editing there, also now `:active` styles should work better
- SocialButton ([ORBIT-1992](https://jira.kiwi.com/browse/ORBIT-1992))
- Popover ([ORBIT-1984](https://jira.kiwi.com/browse/ORBIT-1984))
  - bottom corners weren't rounded when actions were present because of the actions' white background, one solution was to set `overflow: hidden` on the entire Popover, but I didn't want some unintended consequences so I went for the safer option of rounding corners of the actions container
- Separator ([ORBIT-1987](https://jira.kiwi.com/browse/ORBIT-1987))

The two remaining ones are ButtonMobileStore and Card, but those aren't cleared up yet so we'll do them in another PR.

I also refactored CardWrapper as a preparation for UI fixes in Card, so I'm including that in this PR.

 Storybook: https://orbit-silvenon-fix-ui-fixes.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/Separator/index.jsx [hunk 1]
```diff
@@ -1,29 +1,69 @@
...
+      padding-${right}: ${getIndentAmount};
+    `};
+    ${align === "right" &&
+    css`
+      padding-${left}: ${getIndentAmount};
+    `};
+    ${align === "center" &&
+    css`
+      padding-${left}: ${getIndentAmount};
+      padding-${right}: ${getIndentAmount};
```
Comment (mainframev) [inline packages/orbit-components/src/Separator/index.jsx:42]:
```suggestion
      padding: 0 ${getIndentAmount};
```

no need for rtl utility here for both sides :)

File: packages/orbit-components/src/Separator/Separator.stories.jsx [hunk 2]
```diff
@@ -25,7 +26,10 @@ export const Playground = (): React.Node => {
     [null, ...Object.values(SPACINGS_AFTER)],
     SPACINGS_AFTER.LARGEST,
   );
-  return <Separator spaceAfter={spaceAfter} />;
+  const align = select("align", ["left", "right", "center"], "left");
```
Comment (mainframev) [inline packages/orbit-components/src/Separator/Separator.stories.jsx:29]: could you also add that to docs? 🙏🏻

File: packages/orbit-components/src/Popover/components/ContentWrapper.jsx [hunk 3]
```diff
@@ -66,6 +66,8 @@ const StyledActions = styled.div`
       position: relative;
       bottom: auto;
       left: auto;
+      border-bottom-left-radius: 3px;
```
Comment (mainframev) [inline packages/orbit-components/src/Popover/components/ContentWrapper.jsx:69]: use `borderRadiusNormal` token here :)

**Review by mainframev — CHANGES_REQUESTED** [review]
The card ticket in jira has poor description, just says remove borders in responsive. You removed only borders from sides, but left on the top and bottom. Is it correct? Because there is no borders in figma, only for section. 🤔

File: packages/orbit-components/src/Separator/Separator.stories.jsx (hunk 2 above)
Comment (silvenon) [reply packages/orbit-components/src/Separator/Separator.stories.jsx:29]: What do you mean? I added `align` prop to `Separator/README.md` already.

**Comment by silvenon** [comment]
> You removed only borders from sides

I only refactored Card, without any design change, because I wasn't sure what I'm supposed to do. The refactor was a preparation step, but I decided to do the actual UI fix in another PR, once it becomes clear what to do exactly.

File: packages/orbit-components/src/Separator/Separator.stories.jsx (hunk 2 above)
Comment (mainframev) [reply packages/orbit-components/src/Separator/Separator.stories.jsx:29]: I meant to add them to our sandbox

**Review by mainframev — APPROVED** [review]

**Pull request merged**